lib/components/NotFound404/NotFound404.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import type { Meta, StoryObj } from '@storybook/react'
+import { NotFound404 } from './NotFound404'
+import { ButtonGroup } from '../ButtonGroup/ButtonGroup'
+
+const meta: Meta<typeof NotFound404> = {
+  title: 'Components/NotFound404',
```

Comment:
Same as previous component and add `/ErrorMessages/` in the title

Reply:
done